server/commands/screen_commands.c

```diff
@@ -559,7 +559,7 @@ screen_del_key_func(Client *c, int argc, char **argv)
 				*to++ = *from++;
 			}
 		}
-		to = '\0';	/* terminates the new keys string...*/
+		*to = '\0';	/* terminates the new keys string...*/
 	}
```

Comment:
I have looked at this code for half an hour now and not yet figured out what it's supposed to do. It predated git, the feature is undocumented, unused in our code base and quite clearly never worked. I say: Let's just rip out this function - nobody is going to miss it.